index.bs

```diff
@@ -656,6 +656,11 @@ The <dfn method for=FileSystemDirectoryHandle>removeEntry(|name|, |options|)</df
   1. If |access| is not "{{PermissionState/granted}}",
      reject |result| with a {{NotAllowedError}} and abort.
 
+  1. Let |lockResult| be the result of [=file entry/lock/take|taking a lock=]
+     with "`exclusive`" on |entry|.
```

Comment:
Note: we need to ensure that the new locking paradigm covers locked directories appropriately - we should not be able to _remove_ a directory if any contained files are locked, but presumably moving a directory with contained files that are locked is okay

Reply:
Filed #137